### DISCUSSION TITLE
French translation as a community translation

### DISCUSSION THREAD

**Discussion by adiati98** [open-sauced/intro#223] [Announcements]
First, we want to give a big shoutout to @CBID2, @geoffreylgv, @Pondy007, @thomasbnt, and @LouisVdwl for your amazing contribution to the French translation! 💙 

Unfortunately, our team won't be able to continue maintaining the translated versions of our [Open Source Education Path](https://intro.opensauced.pizza/#/README) courses.

This means the existing French translation will no longer be on our website as our official translation.

But here's the good news! We encourage you to maintain and keep the translation going as a [community translation](https://github.com/open-sauced/intro/blob/main/contributing/i18n-guidelines.md#2-community-translations). This way, your awesome work won't be wasted, and the community can still benefit from your translation.

The benefits of maintaining the translation are:

- You will have the opportunity to learn to maintain a project and become the maintainer of the French translation, as it will live in your forked repo.
- You can learn and experience collaborating with contributors, nurturing the translation community, and even assembling your own maintainers' team!
- You will be able to learn how to deploy and host a static website on your own.
- You can update the translation faster by reviewing and merging PRs yourself as a maintainer.

If you are interested in maintaining, deploying, and hosting the French translation in your forked repo, you can respond to this discussion. And if you need help to understand what a maintainer do, you can take our [Becoming a Maintainer](https://intro.opensauced.pizza/#/becoming-a-maintainer/README) course.

Thanks again for your stellar contributions! ✨ 
If you have any questions, feel free to reach out. We're happy to help in any way we can.

**Comment by CBID2** [comment]
Thanks for the shout out @adiati98! 😊 @geoffreylgv and @thomasbnt, how about we maintain the French translation together?

**Reply by geoffreylgv** [reply]
Awesome, thanks @adiati98 for the shout-out, this is very great. 💯 @CBID2 I'm definitely down for continue maintaining the French translation.

**Reply by thomasbnt** [reply]
Thanks for the shoutout @adiati98. 💜
I'm in, let's translate together!

**Comment by LouisVdwl** [comment]
Hey @adiati98, thanks !

I would be very happy to continue actively contributing to this project alongside ALL the contributors who wish to participate.

**Comment by adiati98** [comment]
Hey @CBID2, @geoffreylgv, @LouisVdwl, and @thomasbnt,

Thank you so much for your interest in maintaining the French translation as our Community Translations! ✨ 

Heads up:

- We've migrated docsify to [Docusaurus](https://docusaurus.io/). So you need to pull the latest changes, then run `npm ci` to install the dependencies. Run `npm start` to run the project locally.
- You can read the [Deployment](https://docusaurus.io/docs/deployment) chapter on their official website to deploy and host the project yourself.
- Because we don't maintain translated versions of our courses, we don't have "Language" link on the navbar anymore.
- You need to backup the French translation before you update your `main` branch. **Note:** If you forget to do so and need it, I've backed this up in [this branch](https://github.com/adiati98/intro/tree/chore/french-backup) on my forked repository.

What I can suggest to work on the translation:

- You need to decide amongst yourself at whose forked repository this translation will be developed, deployed, and hosted.
- Have a dedicated branch to develop and deploy the translation (_don't_ use the `main` branch).
- Leave all file names as is, but translate the content. This way, you don't need to change the internal links and it'll save you time.
- You have full control on how contributors contribute to your forked repo. So, you might want to create a contributing guidelines to contribute to the translation and put it in the `contributing` folder located on the root.
- Be aware and up to date with the changes from the `upstream` repo.

You can take a look at the [Spanish translation discussion](https://github.com/open-sauced/intro/discussions/190) as they're also working on theirs. 🙂

**Reply by CBID2** [reply]
Hey @geoffreylgv, @thomasbnt, and @LouisVdwl, I don't mind hosting the translation.

**Reply by geoffreylgv** [reply]
Morning, sure thing. I'd love to host it as well tho :) 
But yeah, let's go

**Reply by thomasbnt** [reply]
Goo! 🚀

**Reply by LouisVdwl** [reply]
Maybe we can create an organization ? :)

**Reply by thomasbnt** [reply]
> Maybe we can create an organization ? :)

sounds like a good idea!

**Reply by CBID2** [reply]
> Maybe we can create an organization ? :)

Good idea @LouisVdwl. @adiati98, do you think we can do this?

**Reply by adiati98** [reply]
> > Maybe we can create an organization ? :)
> 
> Good idea @LouisVdwl. @adiati98, do you think we can do this?  

How you'd like to do in working with the translation repo is up to you, as long as you abide by [the Creative Commons license](https://creativecommons.org/licenses/by/4.0/).
